packages/web-components/src/components/cbp-toast/cbp-toast.scss

```diff
@@ -72,11 +72,15 @@ cbp-toast {
 
   .cbp-toast-sidebar{
     display: flex;
+    align-items: center;
     padding: var(--cbp-space-4x);
-    color: var(--cbp-toast-color-icon-sidebar);
     background-color: var(--cbp-toast-color-bg-sidebar);
     border-top-left-radius: 5px;
     border-bottom-left-radius: 5px;
+
+    cbp-icon{
```

Comment:
ideally, you would target the slot by name and set the color and the icon will inherit it (because it defaults to currentColor)

Reply:
updated to style by slot not by cbp-icon